pkg/config/store/fsstore2.go

```diff
@@ -157,59 +151,43 @@ func (s *fsStore2) readFiles() map[Key]*resource {
 
 func (s *fsStore2) checkAndUpdate() {
 	newData := s.readFiles()
-	s.mu.Lock()
```

Comment:
is checkAndUpdate single threaded now ?
Why was this lock required before and not now?

Reply:
The lock has been moved to the memstore, fsstore2 relies on it.

The lock here was for controls of writing in `checkAndUpdate` and reading in `List`, now both are controlled in memstore internally used by fsstore2; the new `Put` and `Delete` below handles the lock internally.